geospaas_adas_viewer/views.py

```diff
@@ -0,0 +1,7 @@
+from geospaas.base_viewer.views import IndexView
+from geospaas_adas_viewer.forms import ADASSearchForm
+
+class adasIndexView(IndexView):
```

Comment:
Class names must be in CamelCase
```suggestion
class AdasIndexView(IndexView):
```